### PR TITLE
Preparing conclusions, summaries for synthesis and absorption by the project

### PR DESCRIPTION
**Pull request by drphilmarshall**
TL;DR: If you finish your 10 question-conclusions by the end of this week, Zeljko will absorb them into a synthesis to be given to the project sims team as they set up their OpSim 4 runs.

Dear @LSSTScienceCollaborations/observing-strategy-white-paper-editors and @LSSTScienceCollaborations/observing-strategy-white-paper-contributors 

Happy New Year! Great to see several of you working on turning your white paper science cases into journal articles - we all wish you the best for submission :-) 

Zeljko, Beth and I met up yesterday and figured out how we can get the v1.0 Observing Strategy white paper out as well. `OpSim 4` is now close to being ready for release, and Zeljko would like to synthesize our science case conclusions to look for opportunities to influence what the project sims team will simulate with the new code - including new special surveys, rolling cadence schemes and so on. The goal is for the project to  be using the new code to make a new suite of simulated cadences this spring, such that we can start analyzing them in the summer. Zeljko can perform this synthesis in the next three weeks, and update the tensions and tradeoffs chapter with some recommendations. 

This means that Zeljko needs our input! Now's your chance, people:

**Science Section Contributors: If you have not already written up your answers to Zeljko's 10 questions in a "Conclusions" section to your science case, and you want him to take your findings into account in his synthesis, please update your latex files by the end of this week.***

**Chapter Editors: Please be ready to finalize the executive summaries you've been writing early next week, based on the completed science section conclusion sections.**

To prepare the ground for this, I have adopted Humna's suggested style for the section conclusions, and implemented throughout the document. If you already wrote your conclusions, they are now nicely formatted as a set of Q&As! If you have not started yet, your conclusions subsection is ready to edit, just uncomment it from the text. The edits I made are in this PR, so you should be able to find your file and see what I did to it. You can also scroll down and see which sections still need conclusions and which ones are done. 

Beth and I will be working on the doc this week too, starting to tidy things up ready for final editing, and figuring out how the Science Advisory Committee is going to absorb our inputs on Observing Strategy and carry them forward, so you might see some issues on that flying around.

Thanks for all your work so far, and your patience during a quiet autumn! :-)

Best wishes,

Phil

Science sections that still need to have their 10 Q&A Conclusions entered:
```
   AGN/AGN_BELR.tex
   AGN/AGN_Clustering.tex
   AGN/AGN_Disk_Intrinsic.tex
   Cosmology/lenstimedelays.tex
   Cosmology/photoz.tex
   Cosmology/supernovacosmology.tex
   MagellanicClouds/MCs_Exoplanets.tex
   MagellanicClouds/MCs_ProperMotion.tex
   MilkyWay/MW_Astrometry.tex
   MilkyWay/MW_Disk.tex
   MilkyWay/MW_Dust.tex
   MilkyWay/MW_Halo.tex
   MilkyWay/MW_SFH.tex
   SolarSystem/SolarSystem_AsteroidColors.tex
   SolarSystem/SolarSystem_AsteroidLightCurves.tex
   SolarSystem/SolarSystem_CometActivity.tex
   SolarSystem/SolarSystem_Discovery.tex
   SolarSystem/SolarSystem_OrbitalAccuracy.tex
   SolarSystem/SolarSystem_PHA.tex
   SpecialSurveys/M67_special.tex
   SpecialSurveys/magcloudsurvey.tex
   SpecialSurveys/short_exposures.tex
   SpecialSurveys/solar_system_specials.tex
   Transients/cv.tex
   Transients/eruptive.tex
   Transients/grb.tex
   Transients/gw.tex
   Transients/sn.tex
   Transients/tde.tex
   Transients/transientAge.tex
   Variables/cepheids.tex
   Variables/multiperiodicpulsators.tex
   Variables/periodicpulsators.tex
   Variables/planets.tex
   Variables/rotationalvariables.tex
   Variables/youngstars.tex
```

Science sections that have Conclusions written - thank you, authors! :-)
```
   AGN/AGN_Census.tex
   AGN/AGN_Disk_Extrinsic.tex
   Cosmology/lss.tex
   Cosmology/wl.tex
   WFIRST/WFIRST_microlensing.tex
   WFIRST/WFIRST_supernovae.tex
   WFIRST/WFIRST_weaklensing.tex
```

### CODE REVIEW

**Comment by AstroVPK** [comment]
Hi Dr Marshall,
    Can you list the 10 Questions?

**Comment by drphilmarshall** [comment]
Vishal, I've done better than that: they are already in your latex file,
ready to be answered! No need to waste keystrokes on issue threads when you
could be providing initial response in tex and then submitting them for
review by your colleagues as a pull request!

Good luck - and have fun!

**Comment by sschmidt23** [comment]
Whoops @MelissaGraham and I (mostly Melissa) answered these for photoz back in August, but apparently never pasted them into the document.  One of us will do that asap.

**Comment by drphilmarshall** [comment]
Thanks, Sam! Keep those pull requests coming, team! Let's give Zeljko
something to get his teeth into next week :-)

**Comment by AstroVPK** [comment]
Hi Dr Marshall,
     I've just created a pull request for you on the main repository - I've
added responses to the 10 questions. Please confirm that we are all set at
your convenience.

Thanks,
Vishal

            Vishal Kasliwal
 Dept of Physics & Astronomy,
The University of Pennsylvania
                    and
Dept of Astrophysical Sciences,
        Princeton University

**Comment by drphilmarshall** [comment]
Yep, I just merged the PR, Vishal. Thank you!

Don't miss the boat, people! Several chapters are still missing their
sections' conclusions entirely... I guess in the absence of any new
information from you, Zeljko will just continue to use his previous
assumptions. So if you have concerns you want addressed, please do check in!

**Comment by phartigan** [comment]
Dear Phil,

The YSO section is ready, and I believe I have submitted my pull
request correctly, but please let me know if I did not.

Pat Hartigan

Quoting Phil Marshall <notifications@github.com>:

**Comment by drphilmarshall** [comment]
Hi all -

Thanks for all your inputs last week! The science sections in AGN, Milky Way, cosmology and WFIRST now have almost complete conclusion Q&As: would you mind starting there as you pull things together, @ivezic? Variables, Solar System, Transients and Special Surveys/LMCs editors, please try and get any sets of 10 answers that you want to be considered checked in this week, so that Zeljko can include them in his synthesis. Thanks! Looking forward to seeing the distillation that you come up with, Zeljko :-)

Phil

**Comment by ivezic** [comment]
Roger that! 
I wanted to double check that I see these new contributions and I just git 
pull-ed and compiled on master (282 pages in resulting pdf file). However,
I don't see this new text. Am I on wrong branch? If it is indeed master, can
you please point me to an example of the new text to look at?

**Comment by drphilmarshall** [comment]
@ivezic The new text is in the master branch, but to find it you need to step through each science section. As of today there are a couple of sections in the Variables chapter that have their Q&A's complete, but I'd start with Milky Way, AGN, Cosmology and WFIRST before approaching Transients, Variables and Solar System.

Chapter Editors @LSSTScienceCollaborations/observing-strategy-white-paper-editors - it occurs to me that Zeljko might be expecting to see completed executive summaries at the start of your chapters, that pull out the highlights of your chapters' sections' conclusions. How's that effort going? It should be a straightforward precis/abstraction task once your sections are in place.
